examples/sample-input-byo.tfvars

```diff
@@ -50,6 +50,7 @@ storage_type                            = "standard"
 node_pools = {
   cas = {
     "vm_type" = "m5.2xlarge"
+    "cpu_type" = "AL2_x86_64"
```

Comment:
This new `cpu_type` works for ANY vm instance in ANY region? Just checking given this will be the default has to work in all cases. What testing has been done to verify this?

Reply:
Yes, according to AWS docs - https://docs.aws.amazon.com/eks/latest/userguide/create-managed-node-group.html

Reply:
No coding changes just making sure things work in general and that things have been tested. If they have then it's all good.